src/MobileDevice/widget/MobileDevice.ts

```diff
@@ -72,6 +79,8 @@ class MobileDevice extends WidgetBase {
         let errorMessage = "";
         if (this.onDeviceReadyAction === "callMicroflow" && !this.microflow) {
             errorMessage = "on click microflow is required in the 'Events' tab, 'Microflow' property";
+        } else if (this.onDeviceReadyAction === "callNanoflow" && !this.nanoflow.nanoflow) {
+            errorMessage = "on click nanoflow is required in the 'Events' tab, 'Nanoflow' property";
```

Comment:
Error message incorrect, they are no on click ben device ready event, addust also for the the page an mf messages

Reply:
Done